Changelog.md

```diff
@@ -24,8 +24,8 @@ Bugfixes:
  * IR Generator: Add missing cleanup for indexed event arguments of value type.
  * IR Generator: Fix internal error when copying reference types in calldata and storage to struct or array members in memory.
  * IR Generator: Fix IR syntax error when copying storage arrays of structs containing functions.
- * Natspec: Fix ICE when overriding a struct getter with a Natspec-documented return value and the name in the struct is different.
- * TypeChecker: Fix ICE when a constant variable declaration forward references a struct.
+ * Natspec: Fix internal error when overriding a struct getter with a Natspec-documented return value and the name in the struct is different.
+ * Type Checker: Fix internal error when a constant variable declaration forward references a struct.
  * Yul EVM Code Transform: Improved stack shuffling in corner cases.
 
```

Comment:
Could also add the missing newline here. But I also have that in https://github.com/ethereum/solidity/pull/12667...